developerportal/apps/common/test_helpers.py

```diff
@@ -0,0 +1,17 @@
+# Patch wagtail.tests.utils.WagtailPageTests to re-configure use of the ModelBackend
+# so that WagtailPageTests.client.login() will work
+
+from django.test import override_settings
+
+from wagtail.tests.utils import WagtailPageTests
+
+
+class PatchedWagtailPageTests(WagtailPageTests):
+    @override_settings(
+        AUTHENTICATION_BACKENDS=(
+            "mozilla_django_oidc.auth.OIDCAuthenticationBackend",
+            "django.contrib.auth.backends.ModelBackend",
+        )
+    )
+    def login(self):
+        return super().login()
```

Comment:
This approach - to patch back in the `ModelBackend` temporarily - is the least painful/best-fit way I could come up with, given that there's no mock/dummy included in the OIDC authentication backend and I didn't have time to write one. 

Reviewers: let me know if it makes your eye twitch

Reply:
It's gnarly, but it's straightforward and contained.

Add a comment around `developerportal/settings/base.py#L120` noting that backends are overridden in tests to help with potential future changes to `AUTHENTICATION_BACKENDS`?